### DISCUSSION TITLE
Error when import postgrest-kt in Android Studio Project

### DISCUSSION THREAD

**Discussion by lmick002** [supabase/supabase#789] [Questions]
Im attempting to add the postgrest-kt package to my project with the implementation below: 

`    def supabase_version = "0.2.0"
    implementation "io.supabase:postgrest-kt:$supabase_version"`

But I get this build error here: 

`Execution failed for task ':app:dataBindingMergeDependencyArtifactsDebug'.
> Could not resolve all files for configuration ':app:debugCompileClasspath'.
   > Could not find io.supabase:postgrest-kt:0.2.0.
     Required by:
         project :app`

I'm currently running Android Studio 4.2 Beta 5. I'm kind of an Android noob, but any ideas on what could be the issue? Thanks

**Comment by kiwicopple** [comment]
Hey @lmick002 can you please open an issue in the postgrest-kt repo? the `kt` libraries are completely community driven, so please be kind & patient to the maintainers there!

**Reply by lmick002** [reply]
@kiwicopple will do. thx
